cli/operator/node.go

```diff
@@ -116,6 +121,7 @@ var StartNodeCmd = &cobra.Command{
 		if err := operatorNode.StartEth1(eth1.HexStringToSyncOffset(cfg.ETH1Options.ETH1SyncOffset)); err != nil {
 			Logger.Fatal("failed to start eth1", zap.Error(err))
 		}
+		go startMetricsHandler(Logger, cfg)
```

Comment:
@amir-blox lets add a way to disable metrics, could be that if port is not provided(default ) don't collect metrics